processing/src/main/java/org/apache/carbondata/processing/loading/model/CarbonLoadModel.java

```diff
@@ -631,7 +651,7 @@ public void setFactTimeStamp(long factTimeStamp) {
   }
 
   public String[] getDelimiters() {
-    return new String[] { complexDelimiterLevel1, complexDelimiterLevel2 };
+    return new String[] { complexDelimiterLevel1, complexDelimiterLevel2, complexDelimiterLevel3 };
```

Comment:
Modify this class and related changes in other classes as per the above comment

Reply:
done